drud/ddev
# Added DB_CHARSET and DB_COLLATE constants to the wp-config.php

## The Problem/Issue/Bug:

closes #3767.

When non-latin texts (in my case Japanese characters) are saved into the database, it is corrupted/garbled.

This is due to wp-config-ddev.php created during ddev config --project-type=wordpress is missing the following two constants.
```

/** Database charset to use in creating database tables. */
define( 'DB_CHARSET', 'utf8' );

/** The database collate type. Don't change this if in doubt. */
define( 'DB_COLLATE', '' );
```

These two, particularly the DB_CHARSET, when missing, WordPress will save into DB in Latin and not utf8.

## How this PR Solves The Problem:
Added the constant to the wp-config-ddev.php template file

## Manual Testing Instructions:
Run `ddev config --project-type=wordpress` with this branch build and confirm the wp-config-ddev.php file contained definition constants for DB_CHARSET and DB_COLLATE

## Related Issue Link(s):
https://github.com/drud/ddev/issues/3767

## Release/Deployment notes:
Not that I know of.



<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3768"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
